spec/game_spec.rb

```diff
@@ -0,0 +1,22 @@
+# require 'game'
+#
+# describe Rps do
+#
+#   it 'paper beats rock' do
```

Comment:
This looks like a really great start with testing - is there a reason why this is commented out?